include/items.php

```diff
@@ -763,6 +764,17 @@ function item_store($arr,$force_parent = false, $notify = false, $dontcache = fa
 		return 0;
 	}
 
+	// Check for already added items.
+	// There is a timing issue here that sometimes creates double postings.
+	// An unique index would help - but the limitations of MySQL (maximum size of index values) prevent this.
```

Comment:
This really could be a very complex problem since we have to respect all the different installations out there. Maybe we should use "varbinary" as field type? https://dev.mysql.com/doc/refman/5.7/en/binary-varbinary.html

We have to check pro and con - but of course not in this pull request.